.github/workflows/go.yml

```
@@ -18,11 +15,21 @@ jobs:
       with:
         go-version: 1.21
 
-    - name: Build
-      run: go build -v ./...
-
     - name: Test
-      run: go test ./...
+      run: make test
 
     - name: Benchmark
       run: go test -v -run=XXX -benchmem -bench=. ./...
```

might as well make this one:

```suggestion
      run: make benchmark
```

---

oh duh